data/moves.js

```diff
@@ -11324,15 +11324,15 @@ exports.BattleMovedex = {
 			onLockMove: 'rollout',
 			onStart: function () {
 				this.effectData.hitCount = 1;
+				this.effectData.trueDuration = 5;
```

Comment:
Don't call it `trueDuration` if it's not a turn count. Call it `hits` or whatever else it's actually counting.